@here/harp-examples/src/datasource_features_polygons.ts

```diff
@@ -281,7 +281,7 @@ export namespace PolygonsFeaturesExample {
         const mapView = new MapView({
             canvas,
             projection: sphereProjection,
-            clipPlanesEvaluator: new TopViewClipPlanesEvaluator(300000),
+            clipPlanesEvaluator: new TiltViewClipPlanesEvaluator(300000),
```

Comment:
This may be removed if buildings (constructions) height will be embedded/evaluated in DataSource. Then datasource (in this example) should receive its defaultMaxHeight = 300000.